packages/core/src/runner/helper.ts

```diff
@@ -37,29 +39,29 @@ export function getLog(pipcookRunner: PipcookRunner): any {
  * @param self: the target update runner
  * @param saveModelCallback
  */
-export async function assignLatestResult(updatedType: string, result: any, self: PipcookRunner, saveModelCallback?: Function) {
+export async function assignLatestResult(updatedType: string, result: PipcookComponentOutput, self: PipcookRunner, saveModelCallback?: Function) {
```

Comment:
How about make 'updatedType' as a new type contains DATA, MODEL, EVALUATE etc. ?

Reply:
IMO 'saveModelCallback' cloud be a specify type too.

Reply:
IMO It depends on @liangzr, It's acceptable without the change and I'm +1 to refine the in a new PR :)

Reply:
Great.

Reply:
Actually I did want to change the type of `updatedType` to [const **literal** enums](https://www.typescriptlang.org/docs/handbook/enums.html#const-enums). Considering that its original type can at least work, I didn't modify it in this PR